# Stat Homework pg 492 \# 15, 19, 24, 29

In [1]:
from statistics import *
import scipy.stats as st
from math import * 
import matplotlib.pyplot as plt

## \# 15

### A

In [13]:
n = 1001
p = 0.42
invz = st.norm.ppf(.995)

In [14]:
SE = invz * sqrt(.42 * (1-.42) / 1001)
print(SE)

0.0401826600145


$.42 \pm .0402$

### B

no weather reports can be more or less reliable

## \# 19

### A

In [18]:
n = 526
p = 137/n
print(p)
invz = st.norm.ppf(.975)
print(invz)

0.26045627376425856
1.95996398454


In [17]:
SE = invz * sqrt(p*(1-p)/n)
print(SE)

0.0375063290138


$0.26045627376425856 \pm 0.0375063290138$

### B

The confidence interval is tighter .9 vs .95, and the standard error is smaller because the probability is smaller for emails

## \# 24

### A

In [20]:
n = 3000
p = .1
invz = st.norm.ppf(.95)
print(invz)

1.64485362695


In [21]:
sigma = sqrt(p*(1-p)/n)
print(sigma)

0.005477225575051661


In [22]:
print(sigma * invz)

0.00900923435276


$.1 \pm 0.009009$

### B

$n * \pi$ and $n * (1-\pi) >= 10$

population must be >= 10*n so 30,000 or more

random unbiased sampling

## \# 29

In [27]:
n = .25 / (.05/1.95996)**2

print(n)

384.14432015999995


Round up to 385